Makefile

```diff
@@ -26,3 +26,4 @@ clean-deps:
 	$(MAKE) -C reclaimspace $@
 	$(MAKE) -C replication $@
 	$(MAKE) -C volumegroup $@
+	$(MAKE) -C encryptionkeyrotation $@
```

Comment:
please keep this list alphabetically sorted

Reply:
Ack.